parse/MonsterFleetPlansParser.cpp

```diff
@@ -31,8 +31,9 @@ namespace {
 #endif
 
         MonsterFleetPlan* operator()(const std::string& fleet_name, const std::vector<std::string>& ship_design_names,
-                                     double spawn_rate, int spawn_limit, Condition::ConditionBase* location) const
-        { return new MonsterFleetPlan(fleet_name, ship_design_names, spawn_rate, spawn_limit, location); }
+                                     double spawn_rate, int spawn_limit, bool can_alter_starlanes,
+                                     Condition::ConditionBase* location) const
+        { return new MonsterFleetPlan(fleet_name, ship_design_names, spawn_rate, spawn_limit, location, can_alter_starlanes); }
```

Comment:
No, you need the entire package, and anyway particularly this part is something I hope @LGM-Doyle agrees to implement differently.